applications/serial_lte_modem/src/slm_at_commands.c

```diff
@@ -79,7 +78,7 @@ enum sleep_modes {
 };
 
 /** @brief AT command handler type. */
-typedef int (*slm_at_handler_t) (enum at_cmd_type);
+typedef int (*slm_at_handler_t) (enum at_parser_cmd_type);
```

Comment:
This is a leftover from AT_CMD_CUSTOM change, it is not used. Feel free to remove.

Reply:
Removed.